app/src/state/reducers/activity.ts

```diff
@@ -169,6 +169,14 @@ function createActivityReducer() {
         draftState.failCode = action.payload?.status ?? null;
       } else if (Activity.setErrors.match(action)) {
         draftState.activityErrors = getCustomErrorTransformer()(action.payload ?? []);
+      } else if (Activity.updateGeoFailure.match(action)) {
+        draftState.activity.geometry = action.payload.geometry;
+        draftState.activity.form_data.activity_data.latitude = null;
+        draftState.activity.form_data.activity_data.longitude = null;
+        draftState.activity.form_data.activity_data.utm_zone = null;
+        draftState.activity.form_data.activity_data.utm_easting = null;
+        draftState.activity.form_data.activity_data.utm_northing = null;
+        draftState.activity.form_data.activity_data.reported_area = null;
```

Comment:
```suggestion
        draftState.activity.form_data.activity_data.latitude = undefined;
        draftState.activity.form_data.activity_data.longitude = undefined;
        draftState.activity.form_data.activity_data.utm_zone = undefined;
        draftState.activity.form_data.activity_data.utm_easting = undefined;
        draftState.activity.form_data.activity_data.utm_northing = undefined;
        draftState.activity.form_data.activity_data.reported_area = undefined;
```
RJSF thinks Null is a value, and uses undefined to determine empty fields... Ask me how I know :')

When undefined
![image](https://github.com/user-attachments/assets/fafc3dc2-602d-4090-ba40-e3c33b993ee5)

When null
![image](https://github.com/user-attachments/assets/211ca6b5-bdba-4964-8567-89c2f8b9d777)

Reply:
haha, good to know